operations/network/dns/Manage_the_DNS_Unbound_Resolver.md

```diff
@@ -150,3 +150,76 @@ ncn-w001# kubectl -n services patch configmaps cray-dns-unbound \
 --type merge -p '{"binaryData":{"records.json.gz":"H4sICLQ/Z2AAA3JlY29yZHMuanNvbgCLjuUCAETSaHADAAAA"}}'
 ```
 
+### Change the site DNS server
+
+Use the following procedure to change the site DNS server that Unbound forwards queries to.
+
+1. Edit the `cray-dns-unbound` ConfigMap.
+
+  ```bash
```

Comment:
General note - All of the code blocks and text under each step need to be indented to line up with the first letter of the step. They are not rendering properly in the current format.

Reply:
Corrected formatting issues